pkg/services/sqlstore/org_test.go

```diff
@@ -317,6 +317,38 @@ func TestAccountDataAccess(t *testing.T) {
 						})
 					})
 				})
+
+			})
+		})
+
+		Convey("Given multiple saved users", func() {
+			setting.AutoAssignOrg = true
+			setting.AutoAssignOrgId = 1
+			setting.AutoAssignOrgRole = "Viewer"
```

Comment:
You need to clean up after changing global variables, to not bleed into other tests:
```suggestion
			origAutoAssignOrg = setting.AutoAssignOrg
			origAutoAssignOrgId := setting.AutoAssignOrgId
			origAutoAssignOrgRole := setting.AutoAssignOrgRole
			t.Cleanup(func() {
				setting.AutoAssignOrg = origAutoAssignOrg
				setting.AutoAssignOrgId = origAutoAssignOrgId
				setting.AutoAssignOrgRole = origAutoAssignOrgRole
			})
			setting.AutoAssignOrg = true
			setting.AutoAssignOrgId = 1
			setting.AutoAssignOrgRole = "Viewer"
```